ports/mimxrt10xx/linking/flash/AT25SF128A.ld

```diff
@@ -0,0 +1 @@
+_ld_flash_size = 16M;
```

Comment:
All these files feel like not exactly tied to the imx port, could we move them to a common space somewhere?

Reply:
I'd rather leave them here for now. I think there is more rework todo around flash config settings and would rather leave this here until we rework it more broadly. Consider it temporary for now.